test/fixtures/drivers.yml

```diff
@@ -0,0 +1,15 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+  phone: MyString
+  pass: MyString
+  auto: MyString
+  status: MyString
+
+two:
+  name: MyString
+  phone: MyString
```

Comment:
This fixture won't work accordingly to https://github.com/rakvium/taxi-api/pull/18/files#diff-a64c19699f111b607af52963626e2d83R13